src/test/rustdoc/const-display.rs

```diff
@@ -38,10 +39,11 @@ pub const unsafe fn bar_not_gated() -> u32 { 42 }
 pub struct Foo;
 
 impl Foo {
-    // @has 'foo/struct.Foo.html' '//div[@id="method.gated"]/code' 'pub unsafe fn gated() -> u32'
+    // @has 'foo/struct.Foo.html' '//div[@id="method.gated"]/code' 'pub fn gated() -> u32'
+    // @has - '//span[@class="since"]' '1.0.0 (const: unstable)'
     #[stable(feature = "rust1", since = "1.0.0")]
     #[rustc_const_unstable(feature="foo", issue = "none")]
-    pub const unsafe fn gated() -> u32 { 42 }
+    pub const fn gated() -> u32 { 42 }
```

Comment:
Why did this change?

Reply:
In the tests, with `unsafe` it did not render `const` when it is `const_unstable` already. I changed that to test the normal case (a function that was not unsafe). I do not know where the interaction between `unsafe` and `const_unstable` is, but I just wanted to make sure that my changes work well.

Reply:
Ok. Rather than removing the old test, can you add a new test so that both cases are exercised?

Reply:
Or in other words, it's important to know what the interaction between unsafe and const_unstable is/